frontend/src/composables/useShootEditor/helper.js

```diff
@@ -150,106 +158,83 @@ export class EditorCompletions {
     }
 
     forIn(completions, (completion, propertyName) => {
-      let text = generateCompletionText(propertyName, completion.type, token.type)
+      const text = generateCompletionText(propertyName, completion.type, token.type)
       if (token.type === 'firstArrayItem') {
-        text = '- ' + text
+        // text = '- ' + text // TODO: Selection ccurently does not replace the bullet
       }
       const string = propertyName.toLowerCase()
-      const type = generateTypeText(completion.type, completion.format)
+      const typeText = generateTypeText(completion.type, completion.format)
 
       completionArray.push({
-        text,
-        string,
+        label: string,
+        displayLabel: text,
+        detail: typeText,
+        type: 'keyword', // TODO check weather we want to show diffrent icons
```

Comment:
https://codemirror.net/docs/ref/#autocomplete.Completion.type
It is also possible to omit the type property so that no icon is rendered. AFAIK it is not possible to provide a **custom** render function for the completion box (like we did for CM5), only for the content that is shown next to it, the _info_ (details) it is possible to provide a render function

Reply:
let's leave it like it is for now, I do not have a better suggestion